index.html

```diff
@@ -1,75 +1,89 @@
+                <nav>
+                    <a href="index.html">Home</a>
+                    <a href="about.html">About</a>
+                    <a href="#">Products</a>
+                    <a href="#">Blog</a>
+                    <a href="#">Contact</a>
+                </nav>
             </div>
-            <div class="text-container">
-                <h2>The Past</h2>
-                <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
-            </div>
-        </section>
-        
-        <section class="middle-content">
-        
-            <h2>Why Did It Have To Be Boxes...</h2>
+            <img class="jumbo" src="img/jumbo.jpg" alt="Highway Timelapse Photo">
+        </header>
+        <div class="container">
+    
+            <section class="top-content">
+                <div class="text-container">
+                    <h2>The Future</h2>
+                    <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
+                </div>
+                <div class="text-container">
+                    <h2>The Past</h2>
+                    <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
+                </div>
+            </section>
             
-            <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
-            </div>
-        
-        </section>
+            <section class="middle-content">
+            
+                <h2>Why Did It Have To Be Boxes...</h2>
+                
+                <div class="boxes">
+                    <div id="teal" class="box">Box 1</div>
+                    <div id="gold" class="box">Box 2</div>
+                    <div id="cadetblue" class="box">Box 3</div>
+                    <div id="coral" class="box">Box 4</div>
+                    <div id="crimson" class="box">Box 5</div>
+                    <div id="forestgreen" class="box">Box 6</div>
+                    <div id="darkorchid" class="box">Box 7</div>
```

Comment:
Good job an alternate solution would be to add nth-child selectors https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child